LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 Niels Kornerup
```

Comment:
Probably want to update the year to 2016, and license holder to `Infinite Red, Inc.`